### DISCUSSION TITLE
感谢翻译，这个软件有没有设置说明书呀，非常感谢哦。

### DISCUSSION THREAD

**Discussion by liamhui** [zetaloop/ExplorerPatcher#28] [Q&A] [answered]
同上，谢谢。

**Comment by zetaloop** [comment, marked as answer]
原作的百科里有 [使用说明（英文）](https://github.com/valinet/ExplorerPatcher/wiki/All-features)，可打开浏览器自带翻译工具阅读。
也可以在软件设置（“属性”）中直接试试调整各个功能，看看是怎样的效果。

**Reply by zetaloop** [reply]
这个使用说明暂时不打算进行翻译，因为各个功能可以直接试出来是做什么用的。

**Reply by liamhui** [reply]
谢谢你了。软件翻译的已经很好了。 我先根据说明书学习一下吧
